tests/gnrc_ipv6_ext_frag/tests/01-run.py

```diff
@@ -8,6 +8,7 @@
 
 import re
 import os
+import pexpect
 import socket
```

Comment:
import time

Reply:
Fixed. Also cherry-picked #12250 into this PR for now, to better be able to debug https://github.com/RIOT-OS/RIOT/pull/11596#issuecomment-531739735 (will remove it when ACK'd)